seqr/views/utils/terra_api_utils.py

```diff
@@ -67,15 +83,20 @@ def is_anvil_authenticated(user):
 def _get_call_args(path, headers=None, root_url=None):
     if headers is None:
         headers = {"User-Agent": SEQR_USER_AGENT}
-    headers.update({"accept": "application/json"})
+    headers.update({"accept": "application/json", "Content-Type": "application/json"})
     if root_url is None:
         root_url = TERRA_API_ROOT_URL
     url = urljoin(root_url, path)
     return url, headers
 
 
+def _safe_get_social(user):
+    social = user.social_auth.filter(provider=SOCIAL_AUTH_PROVIDER) if hasattr(user, 'social_auth') else []
```

Comment:
This was clearly added because there are times when `user` does not have the social auth attribute. Do we hit both the case when it does and does not in our tests? i.e. if you took this edge case handling out, would a test fail?

Reply:
When opening `workspace/workspace_namespace/workspace_name` without logging in, the `user` will be `AnonymousUser`. The `AnonymousUser` doesn't have the `social_auth` attribute even when the social auth is enabled. We could add protection in `is_google_authenticated()` then there is no need to check the `social_auth` attribute for every user.

Reply:
I think this is fine as is, but if you think it would be cleaner to move it into `is_google_authenticated` thats up to you.  I just wanted to make sure we are testing this. But since we do test those endpoints without logging in it seems that we are